### PR TITLE
Wrapper for TempProbe

### PR DESCRIPTION
**Pull request by cherbel**
Update: We have clean test runs.

~~This does not work due to the issue submitted here: https://github.com/Arduino-CI/arduino_ci/issues/192~~

### CODE REVIEW

**Comment by jgfoster** [comment]
We are also waiting on https://github.com/Arduino-CI/arduino_ci/issues/193.

**Comment by ianfixes** [comment]
I think the problem is elsewhere, see https://github.com/cherbel/TankControllerLib/pull/1

**Comment by ianfixes** [comment]
It looks like you might be [checking out the wrong branch](https://github.com/Open-Acidification/TankControllerLib/blob/master/scripts/install_libraries.sh#L19) in your install script:

```bash
# get custom version of Adafruit_MAX31865
git clone https://github.com/Arduino-CI/Adafruit_MAX31865.git
mv Adafruit_MAX31865 $(bundle exec arduino_library_location.rb)
```

The [error you're getting about `Adafruit_MAX31865_CI.h: No such file or directory `](https://github.com/Open-Acidification/TankControllerLib/runs/1398445342#step:6:64) makes sense because the `Adafruit_MAX31865_CI.h` file is located in [the `arduino_ci` branch of `Arduino-CI/Adafruit_MAX31865`](https://github.com/Arduino-CI/Adafruit_MAX31865/tree/arduino_ci/src), and in the snippet above you're just cloning the master branch.

**Comment by jgfoster** [comment]
See https://github.com/Arduino-CI/arduino_ci/pull/212 for an example of what we are trying to do.

**Comment by ianfixes** [comment]
I just merged `arduino_ci` code that should fix this issue

**Comment by cherbel** [comment]
@jgfoster I think this is ready to be merged.

File: src/Devices/TempProbe_TC.h [hunk 1]
```diff
@@ -0,0 +1,116 @@
+// Adafruit MAX31865
```
Comment (prestoncarman) [inline src/Devices/TempProbe_TC.h]: Please convert this to a multi-line comment to allow it to show in documentation.

**Review by prestoncarman — CHANGES_REQUESTED** [review]

**Comment by prestoncarman** [comment]
If this is ready, please update the PR title and description.

**Review by prestoncarman — APPROVED** [review]
LGTM

**Comment by prestoncarman** [comment]
@all-contributors please add @cherbel for code

**Pull request merged**